packages/fast-development-site-react/src/components/site/component-view-toggle.tsx

```diff
@@ -2,30 +2,44 @@ import * as React from "react";
 import { toPx } from "@microsoft/fast-jss-utilities";
 import { Link, withRouter } from "react-router-dom";
 import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
-import { IDevSiteDesignSystem } from "../design-system";
+import devSiteDesignSystemDefaults, { IDevSiteDesignSystem } from "../design-system";
 
 export interface IComponentViewToggleClassNameContract {
     componentViewToggle: string;
 }
 const styles: ComponentStyles<IComponentViewToggleClassNameContract, IDevSiteDesignSystem> = {
     componentViewToggle: {
         width: toPx(40),
-        height: toPx(40),
+        height: toPx(38),
         display: "flex",
         position: "relative",
         alignItems: "center",
+        border: `${toPx(1)} solid transparent`,
         justifyContent: "center",
+        borderRadius: toPx(3),
```

Comment:
Every other instance of border-radius we have is 2px - is this an intentional change to 3px?

Reply:
Confirmed we are using 2px throughout. Can you update these to be 2px?